packages/blockchain-wallet-v4-frontend/src/modals/TransactionReport/services.js

```diff
@@ -1,29 +1,33 @@
-import moment from 'moment'
+import moment from 'services/MomentHelper'
 
 export const isDatePosteriorToBtcGenesis = selectedDate => {
```

Comment:
All the functions isSomething(Btc/Bch)Something(...) should be more generic (eg: isSomethingSomething(..., coin) ) because we don't want to add a new function everytime we add a coin.

Reply:
Mmm I understand the point, but if the function was generic, it would go in the FormHelper instead of staying here, we'll have to factorize more the validation rules once we reached some consensus/conventions on how to pass the props to our form.

Reply:
For the comment below regarding the mobile value, no its basically in bytes: 10000 which is 32